vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/RemoteFileManager.kt

```diff
@@ -0,0 +1,33 @@
+package com.malinskiy.marathon.android
+
+import com.malinskiy.marathon.test.Test
+
+class RemoteFileManager(private val device: AndroidDevice) {
```

Comment:
Maybe will be better to extract interface here?

Reply:
I don't see this becoming standard between vendor modules for other platforms. For Android it's already using the AndroidDevice as an abstraction.
What would be the purpose of this?

Reply:
I'm agree with @Malinskiy. We already have abstraction here